disperser/dataapi/server.go

```diff
@@ -69,6 +69,12 @@ type (
 		Data []*BlobMetadataResponse `json:"data"`
 	}
 
+	UnsignedBatches struct {
```

Comment:
We actually need this:
- `TotalUnsignedBatches` per operator: as the name indicates, we need to know how many unsigned batches each operator has (we interest in those who have greater than 0 unsigned batches)
- `TotalBatches`: this is what you already defined

With the above two, what we want to get is that for each operator, what's the percentage of their unsigned batches. This is `percent[operator] = TotalUnsignedBatches[operator] / TotalBatches`.